src/content/contributing/translation-program/faq/index.md

```diff
@@ -9,6 +9,18 @@ sidebar: true
 
 If you're new to the Translation Program and are hesitant to jump in, here are some FAQs that can help you begin. Use this guide to find answers to the most common queries.
 
+## Can I get compensated for translating ethereum.org? {#compensation}
+
+Ethereum.org is an open source website, which means that anyone can get involved and contribute.
+
+The ethereum.org Translation Program is an extention of that and is organized with a similar philosophy in mind.
+
+The aim of the Translation Program is to localize the website and make Ethereum content accessible to everyone, regardless of their language, but also to allow anyone who is bilingual to get involved with the Ethereum ecosystem and contribute in an accessible way.
+
+For this reason, the Translation Program is open and voluntary, and participation is not subject to compensation. If we decided to pay translators for their work on the website, we would only be able to accept professional translators with sufficient experience, and add other restrictions to who can participate in the Translation Program. This would make the Translation Program exclusionary and prevent us from reaching the goals, outlined above, specifically to allow everyone to participate and get involved with the ecosystem.
```

Comment:
```suggestion
For this reason, the Translation Program is open and voluntary, and participation is not subject to compensation. If we were to compensate translators for the number of words they translate, we could only invite those with sufficient translation experience (professional translators) to join the translation program. This would make the Translation Program exclusionary and prevent us from reaching the outlined goals, specifically: allowing everyone to participate and get involved with the ecosystem.
```